### PR TITLE
feat: Updating the corresponding dashboard and documentation

### PR DESCRIPTION
**Pull request by zhengjianhui**
1. Updating the corresponding dashboard and documentation
2. Query limit configuration for loki

### CODE REVIEW

File: apm/docs/metrics.md [hunk 1]
```diff
@@ -671,6 +673,15 @@ Consensus time for P90
 ```
 avg(histogram_quantile(0.90, sum(rate(muta_consensus_duration_seconds_bucket[5m])) by (le, instance)))
 ```
+
+##### /
+Not show, for alert
+```
+avg(histogram_quantile(0.90, sum(rate(muta_consensus_duration_seconds_bucket[5m])) by (le, instance))) / avg(histogram_quantile(0.90, sum(rate(muta_consensus_time_cost_seconds_bucket{type="exec"}[5m])) by (le, instance)))
```
Comment (homura) [inline apm/docs/metrics.md:680]: Would you explain what this expression does?

Comment (zhengjianhui) [reply apm/docs/metrics.md:680]: Used to determine if the consensus time is greater than the exec time

Comment (homura) [reply apm/docs/metrics.md:680]: So it alerts when the ratio exceeds 1, right?  It would be better to mark this result in the document

Comment (zhengjianhui) [reply apm/docs/metrics.md:680]: Yeah, this is going to move into the alert table file.

**Pull request merged**